src/app/ticker/TickerPage.js

```diff
@@ -16,73 +17,116 @@ import {
 } from 'carbon-components-react'
 
 import TickerForm, { formName } from './components/TickerForm'
-import { reserve, expiryLimit } from './actions'
+import { reserve, expiryLimit, faucet, getPolyFee } from './actions'
 import { data as tokenData } from '../token/actions'
 
 type StateProps = {|
   account: ?string,
   token: Object,
-  expiryLimit: number,
+  networkName: string,
+  polyBalance: BigNumber,
+  expiryLimit: number
 |}
 
 type DispatchProps = {|
-  change: (?string) => any,
-  reserve: () => any,
+  change: (? string) => any,
```

Comment:
`? string` > `?string`
see you even above have different code style...
Again: if you want to change code style, we should discuss it separately